src/components/search-bar/SearchbarComponent.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="q-mr-md search-container">
+    <q-input 
+      rounded 
+      dense 
+      outlined 
+      v-model="searchText" 
+      placeholder="Search"
+      bg-color="grey-2"
+      clearable
+      @update:model-value="search"
+      >
+      <template v-slot:append>
+        <q-icon name="search" />
+      </template>
+    </q-input>
+
+    <q-list 
+      v-if="showSuggestions && searchText"  
+      class="suggestions-list"
+    >
+      <q-item 
+        v-for="profile in filteredProfiles" 
+        :key="profile.key" 
+        clickable 
+        @click="selectProfile(profile)"
+        rounded
+      >
+        <q-item-section avatar>
+          <q-avatar class="avatar">
+            <img :src="`https://cdn.quasar.dev/img/${profile.avatar}`">
```

Comment:
I thought profile Avatar in production is absolute path... In emulator it's empty by default.
```suggestion
            <img :src="`${profile.avatar}`">
```